sd_webui_bayesian_merger/merger.py

```diff
@@ -197,10 +197,13 @@ def merge(
                 merged_model[key] = result[1]
 
         for key in tqdm(thetas["model_b"].keys(), desc="stage 2"):
-            if KEY_POSITION_IDS in key or "model" not in key:
-                continue
-            if key not in merged_model:
-                merged_model.update({key: thetas["model_b"][key]})
+            if "model" in key and key not in merged_model:
+                if KEY_POSITION_IDS in key:
```

Comment:
Is there a point? There isn't any logic currently in the merger that deals with skipping clips, force reseting, or keeping the internal clip logic the same in the models.

Reply:
Indeed, but currently there is no option anywhere for a user to configure that. it's why pass is there, so that we don't iterate through the for loop, but it can be added later.

Reply:
Okay, because in previous descriptions of that configuration variable, you stated that it corresponded to the clip skip value in WebUI, which is different from the clip value in super merger.

Reply:
This has been added back in as of the newest commit.